applications/mne_scan/plugins/plugins.pro

```diff
@@ -59,7 +59,11 @@ contains(MNECPP_CONFIG, buildBasicMneScanVersion) {
         # eegosports \
         # brainamp \
         # tmsi \
-        natus \
+        natus
+
+    USE_LSL {
```

Comment:
Incorporate changes below and then change this line to:
`contains(MNECPP_CONFIG, useLSL) {
SUBDIRS += lsladapter
}`